sentry/src/main/java/io/sentry/ExternalOptions.java

```diff
@@ -254,8 +275,19 @@ public void addInAppExclude(final @NotNull String exclude) {
     inAppExcludes.add(exclude);
   }
 
+  @Deprecated
+  @SuppressWarnings("InlineMeSuggester")
   public void addTracingOrigin(final @NotNull String tracingOrigin) {
-    this.tracingOrigins.add(tracingOrigin);
+    this.addTracePropagationTarget(tracingOrigin);
+  }
+
+  public void addTracePropagationTarget(final @NotNull String tracePropagationTarget) {
+    if (tracePropagationTargets == null) {
+      tracePropagationTargets = new CopyOnWriteArrayList<>();
+    }
```

Comment:
I'd rather create the instance directly in the declaration with the default value.

Reply:
The default value is missing here too.